modules/helfi_node_survey/config/install/field.field.node.survey.field_survey_content_pages.yml

```diff
@@ -0,0 +1,31 @@
+uuid: 9cb9a071-2d67-43d0-a8e3-6e2b7142ee17
```

Comment:
Should the survey content type also have the "publish on all pages" option like the announcement nodes do?

Reply:
I think that is redundant, I'm inferring "publish on all pages" from `field_survey_content_pages`.

https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/773/files#diff-471642b1269fee7b98777d63d4f8d269a9eeeacf5dc67b50026c5cc10f57112cR103-R106